src/XrdMacaroons/XrdMacaroonsAuthz.cc

```diff
@@ -173,6 +173,10 @@ Authz::Access(const XrdSecEntity *Entity, const char *path,
 
     const char *authz = env ? env->Get("authz") : nullptr;
     if (authz && !strncmp(authz, "Bearer%20", 9))
+    {
+        authz += 9;
```

Comment:
If and else have the same action. Something doesn't look right.

Reply:
The desired behavior is correct, the formatting is clumsy.  I'll see if I can't come up with a way to make it readable without having a 150 character conditional.

Reply:
I'm not sure the formatting is all that much better -- but came up with a revision that's hopefully less confusing.